### PR TITLE
Added a note on python hanging issue on Windows

### LINKED ISSUE #175: Windows python3 issue
May want to move the python3 alias instruction up into the main text of setup page, this seemed to be a **super** common issue in our pilot

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #175.

### CODE REVIEW

**Review by steve-crouch — COMMENTED** [review]
Might want to consider working the 'Making sure...' callout into the main text itself, since a callout is sometimes seen as optional - if they choose not to do it, and later encounter the Python invocation problem, they may miss the handy link to the resolution in the callout.

**Comment by anenadic** [comment]
> Might want to consider working the 'Making sure...' callout into the main text itself, since a callout is sometimes seen as optional - if they choose not to do it, and later encounter the Python invocation problem, they may miss the handy link to the resolution in the callout.

I wanted to make it visually distinctive so it is not missed in the large body of text - how about I use the .prereq block instead?

File: _episodes/12-virtual-environments.md [hunk 1]
```diff
@@ -196,6 +196,17 @@ From <a href="https://xkcd.com/1987/" target="_blank">XKCD</a> (Creative Commons
...
+> ## Making Sure You Can Invoke Python
+> You can test your Python installation from the command line with:
+> ~~~
+> $ python3 --version # on Mac/Linux
+> $ python --version # on Windows — Windows installation comes with a python.exe file rather than a python3.exe file 
+> ~~~
+> {: .language-bash}
+> If you are using Windows and invoking `python` command causes your Git Bash terminal to hang with no error message or output, you may 
+need to create an alias for the python executable `python.exe`, as explained in the [troubleshooting section](../common-issues/index.html#python-hangs-in-git-bash).
+{: .callout}
```
Comment (anenadic) [inline _episodes/12-virtual-environments.md:208]:
```suggestion
{: .prereq}
```

**Review by steve-crouch — APPROVED** [review]
Yep, that's a good way to highlight it.

**Pull request merged**